src/services/TokenService.ts

```diff
@@ -71,7 +71,8 @@ export class TokenService {
 
   public async initialize(): Promise<TokenListMap> {
     this.geckoCoinInfo = new Map<string, string>();
-    const uri = `https://pro-api.coingecko.com/api/v3/coins/list?x_cg_pro_api_key=${process.env.COINGECKO_API_KEY}`;
+    this.coinAddressIdDictionary = new Map<string, string>();
+    const uri = `https://pro-api.coingecko.com/api/v3/coins/list?include_platform=true&x_cg_pro_api_key=${process.env.COINGECKO_API_KEY}`;
```

Comment:
Would have expected to have some words about `include_platform` in the initial comment pointing to their api docs
https://www.coingecko.com/en/api/documentation